src/erlbin_notificator_sup.erl

```diff
@@ -0,0 +1,39 @@
+-module(erlbin_notificator_sup).
+
+-behaviour(supervisor).
+
+%% API
+-export([start_link/0]).
+
+%% Supervisor callbacks
+-export([init/1]).
+
+-define(SERVER, ?MODULE).
+
+%%====================================================================
```

Comment:
a lot of people use this big chunks of comment. I hate them, they don't add nothing, do whatever suits better for you. i generally add a % private before all the private functions, but apart from that i don't specify which are gen server and which are public functions.